### PR TITLE
MTDSA-3556 Created Losses RAML boiler plate

### PR DESCRIPTION
**Pull request by narendravijayarao**

### CODE REVIEW

File: resources/public/api/conf/1.0/broughtForwardLosses_create.raml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.authorization
+  - headers.contentTypeJson
+  - errors.notFound
+  - errors.agentSubscription
+  - errors.clientSubscriptionToMTD
+  - errors.formatNino
+  - errors.formatTaxYear
+description: Create a Brought Forward Loss
```
[the remark this answers is not in the document]
Comment (narendravijayarao) [reply resources/public/api/conf/1.0/broughtForwardLosses_create.raml]: this is just a boilerplate... whoever working on the create will write proper description

File: resources/public/api/conf/1.0/broughtForwardLosses_create.raml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
```
[the remark this answers is not in the document]
Comment (narendravijayarao) [reply resources/public/api/conf/1.0/broughtForwardLosses_create.raml]: same as above

[the remark this answers is not in the document]
Comment (james-work-account) [reply resources/public/api/conf/1.0/broughtForwardLosses_create.raml]: This line is in the file twice (line 3 and line 5). Once of these needs removing.

File: resources/public/api/conf/1.0/broughtForwardLosses_create.raml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  - headers.authorization
```
[the remark this answers is not in the document]
Comment (james-work-account) [reply resources/public/api/conf/1.0/broughtForwardLosses_create.raml]: This displays as below:

```
Authorization               An OAuth 2.0 Bearer Token with the write:self-assessment or read:self-assessment scope.
required                    For example: Bearer bb7fed3fe10dd235a2ccda3d50fb
-- | --
Authorization               An OAuth 2.0 Bearer Token with the write:self-assessmentscope.
required                    For example: Bearer bb7fed3fe10dd235a2ccda3d50fb
```
It isn't needed, even in a boilerplate.

**Pull request merged**